static/js/recent_topics_ui.js

```diff
@@ -556,7 +556,7 @@ export function inplace_rerender(topic_key) {
             const rendered_row = render_recent_topic_row(format_conversation(topic_data));
```

Comment:
I think we want to remove the access to `topics_widget.meta.offset` as well, so that can be unexported.

I don't understand why we can't have a function that accepts parameters, possibly including a callback, like `topics_widget.insert_row(rendered_row, topic_insert_index);` ; the last ~10 lines of this block seem like they're only functions of those inputs? We'd end up calling `render_recent_topic_row` unconditionally, is that the concern? We could pass a `lambda` to do that if that was the problem.

Reply:
Not sure what I was thinking (must be my ongoing cold), did the proper refactor. I am looking forward to using these functions when inserting new messages and remove `complete_rerender` from that code path.